pynq_dna/PYNQ_DNA.tcl

```diff
@@ -469,7 +469,9 @@ export_ip_user_files -of_objects [get_files ./src/design_1.bd] -no_script -force
 export_simulation -of_objects [get_files ./src/design_1.bd] -directory ./DNA_extractor/DNA_extractor.ip_user_files/sim_scripts -ip_user_files_dir ./DNA_extractor/DNA_extractor.ip_user_files -ipstatic_source_dir ./DNA_extractor/DNA_extractor.ip_user_files/ipstatic -force -quiet
 launch_runs impl_1 -to_step write_bitstream -jobs 6
 wait_on_run impl_1
-write_bd_tcl -force ./bin/dna_extractor.tcl
+# write_bd_tcl -force ./bin/dna_extractor.tcl  # Start using .hwh file isntead
```

Comment:
Is there a reason to leave this in as a comment rather than just moving this line?